Documentation/release-notes/5111.md

```diff
@@ -0,0 +1,9 @@
+#### Application and library build and deployment
+
+  * [Developer Community 1184573][0]: Xamarin Android falsely
+    saying that Resource does not exist in the current context (CS0103).
+    Duplicate `attr` elements in `declare-styleable`  styles were
+    causing the DesignTime Build system to crash. As a result the
+    `Resource.designer.cs` was not being updated.
+
+[0]: https://developercommunity.visualstudio.com/content/problem/1167082/xamarin-android-falsely-saying-that-resource-does.html
```

Comment:
To facilitate collation into the final release notes, this can inline the devcom link and use the phrase "design-time build":

- [Developer Community 1184573](https://developercommunity.visualstudio.com/content/problem/1167082/xamarin-android-falsely-saying-that-resource-does.html):
  Xamarin Android falsely saying that Resource does not exist in the
  current context (CS0103).  Duplicate `attr` elements in
  `declare-styleable`  styles were causing the design-time build system
  to crash. As a result, the _Resource.designer.cs_ file was not being
  updated.